generation/configs/text.yaml

```diff
@@ -35,9 +35,9 @@ lambda_zero123: 0
 # warmup rgb supervision for image-to-3d
 warmup_rgb_loss: True
 # training batch size per iter
-batch_size: 1
+batch_size: 4
```

Comment:
why 4 not 5?

Reply:
for test purpose only
can ignore changes on this file